### PR TITLE
Tasks-61-70

### PR DESCRIPTION
**Pull request by KamilWasilonek**

### CODE REVIEW

File: tasks/62_swap-cases/swap-cases.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export function swapCases(str) {
+  return [...str]
```
Comment (asmyk) [inline tasks/62_swap-cases/swap-cases.js:2]: why did you use array and map here?

File: tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export function sumFibs(num) {
```
Comment (asmyk) [inline tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js:1]: as follow up to fibo look how you can memoize: https://medium.com/@reallygordon/implementing-memoization-in-javascript-5d140bb04166

File: tasks/69_unpack-elements/unpack-elements.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export function unpackArray(arrToUnpack) {
```
Comment (asmyk) [inline tasks/69_unpack-elements/unpack-elements.js:1]: as followup - what will happen if you get [[[[[[['a']]]]]]] ? how can you prevent this?

**Review by asmyk — APPROVED** [review]

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export function smallestCommons(arr) {
```
Comment (tkachukivan) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js:1]: take a look at Euclidean Algorithm it will easier and probably eat fewer resources

**Review by tkachukivan — APPROVED** [review]

**Pull request merged**